packages/libs/ui/nft-react-hooks/src/lib/hooks/useWalletNFTs/useWalletNFTs.ts

```
@@ -15,8 +15,10 @@ function useWalletNFTs(args: Args) {
   let isSearchValid = false;
 
   if ('ensName' in args) {
+    if (args.ensName?.length > 255) throw new Error('Input too long');
```

not sure if we want to throw an error here or just return null or empty array